service/routes.py

```diff
@@ -33,3 +35,63 @@ def init_db():
     """ Initializes the SQLAlchemy app """
     global app
     Inventory.init_db(app)
+
+@app.route("/products/<product_id>", methods=["GET"])
```

Comment:
Probably the Products team will call it `/products`. Do we want to have something like `inventory-records`?

Reply:
hmm good point I had initially named it inventory-records but thought since we are referring to products we should have products but you are right the products team may have that.
In that case I will rename the functions as well to get-records and create_records instead of using products

Reply:
Done